### PR TITLE
[ADI] Add themes, needs and idea categories list endpoints

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -376,6 +376,21 @@ nelmio_cors:
...
 
 api_platform:
-    mapping:
-        paths: ['%kernel.project_dir%/src/Entity']
-    enable_docs: '%kernel.debug%'
+    name_converter: serializer.name_converter.camel_case_to_snake_case
+    resource_class_directories:
+        - '%kernel.project_dir%/src/Entity'
+    enable_docs: false
+    enable_swagger_ui: false
```
Comment (ottaviano) [inline app/config/config.yml]: I think it is not very useful as you already disable the doc with `enable_docs: false`, WDYT?

File: app/config/config_test.yml [hunk 2]
```diff
@@ -125,3 +125,11 @@ doctrine:
             default:
                 logging: false
                 profiling: false
+
+api_platform:
+    enable_docs: false
```
Comment (ottaviano) [inline app/config/config_test.yml]: not needed as you do it in the global config file

File: src/Entity/IdeasWorkshop/EnabledInterface.php [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace AppBundle\Entity\IdeasWorkshop;
```
Comment (ottaviano) [inline src/Entity/IdeasWorkshop/EnabledInterface.php]: what about moving it to `AppBundle\Entity`? This interface can be used by other entities

File: src/Entity/IdeasWorkshop/EnabledInterface.php [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+
+interface EnabledInterface
+{
```
Comment (ottaviano) [inline src/Entity/IdeasWorkshop/EnabledInterface.php]: what about adding `public function isEnabled(): bool` method?

File: app/config/config_test.yml (hunk 2 above)
Comment (illusionOfParadise) [reply app/config/config_test.yml]: `config_test.yml` extends `config_dev.yml` that changes these values.

File: src/Entity/IdeasWorkshop/EnabledInterface.php (hunk 4 above)
Comment (illusionOfParadise) [reply src/Entity/IdeasWorkshop/EnabledInterface.php]: Done.

File: src/Entity/IdeasWorkshop/EnabledInterface.php (hunk 3 above)
Comment (illusionOfParadise) [reply src/Entity/IdeasWorkshop/EnabledInterface.php]: Done.

File: app/config/config.yml (hunk 1 above)
Comment (illusionOfParadise) [reply app/config/config.yml]: Done.

**Pull request merged**